### ISSUE TITLE
[BUG] Most planets spawn with the same seed. 

### ISSUE THREAD

**Issue by CoasterBlock** [tsunamayo/Starship-EVO#5701]
### Short Description of the issue:

Most planets seem to spawn with the exact same terrain seed.

### Environment
- Starship EVO version: Latest Experimental
- Graphics Card: RTX 2080 Super

### Additional information:

I noticed that most planets spawn with the exact same pattern, the only difference is color and some minor details. There is a massive lack of continental worlds with water. Planets with water are very rare. This same bug also affects moons. Most moons seem to have identical noise pattern, only color is the difference. This affects planets all around the galaxy and isnt limited to a character save. I know planet are WIP, however it seems this bug is more recent. 

![20240407122427_1](https://github.com/tsunamayo/Starship-EVO/assets/126499309/a1f4d3e3-72cb-476a-8229-02f6e568238b)

**Comment by Aranhanauta** [comment]
I noticed almost the same problem.

But it was the case that solar systems generated repetitive sets of planets and moons.

Sometimes two star systems had the same planet and number of moons and even the same type of moon orbiting them, rarely did they have a new planet.

Not to mention that most star systems have few planets (in my opinion), with most of them having just one planet or two, and there are rarely more.

I wanted to know if this could be related to the bug.

**Comment by CoasterBlock** [comment]
@Aranhanauta Good point! I noticed as well! Some more variety would be nice. I know planets are supposed to be refactored after survival comes out, with new climates, biomes etc. But this does seem like a recent bug. I dont recall it being this repetetive

**Comment by tsunamayo** [comment]
@CoasterBlock Hi, can you send me your character save if it is still in that system?
